### ISSUE TITLE
Login with Biometrics is cumbersome

### ISSUE THREAD

**Issue by uncovery** [beemdevelopment/Aegis#1288]
### Version

2.2.2

### Source

Google Play

### Vault encryption

Yes (with biometric unlock)

### Device

Sasumg G23 Ultra

### Android version

14

### Steps to reproduce

I am not sure how to 100% reproduce this, Aegis on my phone has been behaving like this since the start. I originally imported the data from another app. 

### What do you expect to happen?

1. Open Aegis 
2. Prompt for fingerprint appears, unlock the app, use it.

### What happens instead?

1. Open Aegis (the following happens EACH TIME I use the app.
2. See the screen: "[ password field ] [ UNLOCK ] [tap here to use biometrics]"
3. Tap "tap here to use biometrics"
4. Message pops up "Biometric unlock by default. "We occasionally prompt you to enter your password instead to ensure you haven't forgotten it and won't get locked out of your vault. After entering your password once, Aegis will default back to biometric unlock until it's time for another password reminder."
5. Click OK
6. Prompt for fingerprint appears, unlock the app, use it.

**Comment by alexbakker** [comment]
Could you share the full message that's displayed in the dialog that pops up?

**Comment by uncovery** [comment]
hi there, updated it above.

**Comment by alexbakker** [comment]
> After entering your password once, Aegis will default back to biometric unlock until it's time for another password reminder

There ya go.

**Comment by uncovery** [comment]
DUH. ok thanks for the hint. It works now.

For reference, this is how it went on in my head:

1) Aegis never had the "default to biometrics" when I installed it. It always prompted for the password or alternatively push the "use biometrics". so I got used to that second step, assuming the deault to biometric was broken, and patiently waited for future versions to fix this,
2) From some point on it ALSO sent me the prompt. My brain went "oh so now 2 things are broken. It does not default to it and also let's me know that it will eventually ask me for the password, but does not actually ask me for it right now, since biometrics still works." I assumed that if the prompt would "do it's thing", it would actually prevent me from using biometrics.
4) at one point I was wondering that this would not get fixed and came here.

Well, issue solved.
